### ISSUE TITLE
PCA on Softmax implementation

### ISSUE THREAD

**Issue by rodrigoabreu22** [rodrigoabreu22/TAA_American_Sign_Language_Understanding#20]
Implement PCA on softmax implementation to reduce image noise and target better model performance and reduce overfitting.
